lib/gringotts/gateways/monei.ex

```diff
@@ -389,27 +442,40 @@ defmodule Gringotts.Gateways.Monei do
   end
 
   # Makes the request to MONEI's network.
-  @spec commit(atom, String.t(), keyword, map) :: {:ok | :error, Response}
+  @spec commit(atom, String.t(), keyword, keyword) :: {:ok | :error, Response}
```

Comment:
spec for a private method, is it required?

Reply:
It won't appear in the docs anyways. Dialyzer can use it too.